rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,184 @@
+====== PHP RFC: PHP Technical Committee ======
+  * Version: 0.2
+  * Date: 2023-03-03
+  * Author: Jakub Zelenka, bukka@php.net
+  * Status: Draft
+
+===== Introduction =====
+
+PHP uses its voting system to decide about changes to PHP. This was created mainly for user facing changes. It works
+well for that purpose but it is not that effective for deciding about purely technical changes that impact PHP
+internals and extension API's. It is not clear how it should resolve conflicts on the technical basis between
+contributors that sometimes happen.
+
+
+===== Proposal =====
+
+This RFC proposes an introduction of the PHP Technical Committee (TC) that would decide about all technical aspects of
+php-src. Its primary but not only role is to resolve disagreements between contributors.
+
+==== Definitions ====
+
+The text contains various terms that need further clarification to prevent future wording dispute. Here are their
+definitions:
+
+  * php-src - Source code in https://github.com/php/php-src/.
+  * contributor - Anyone who has already contributed to php-src.
+  * core developer - Anyone with write commit access to php-src.
+  * dev branch - php-src branch consisting of PHP- prefix and major and minor version number (e.g. PHP-8.2).
+  * maintainer - person responsible for extension or SAPI that is listed in the php-src EXTENSION file.
+  * issue - php-src GitHub issue, including pull requests.
+  * PR - Pull request
+  * process changes - Any changes impacting the way how PHP project is managed and its bylaws.
```

Comment:
```suggestion
  * process changes - Any changes impacting the way how PHP project and its bylaws are managed.
```

Reply:
But it is also for changes to PHP bylaws and not just how its bylaws are managed, isn't it? Although I'm not sure if the term bylaws is fine here as technically we don't define PHP bylaws anywhere. Maybe @Crell might have an idea how to better define process changes.

Reply:
I wasn't sure about that. Defining bylaws seems sensible, or avoiding the term in the first place.

Reply:
Process RFCs collectively *should* constitute the project's bylaws, but you're right that's not the current wording.  Avoiding that word for the moment until it's defined more formally (similar to how FIG does it, probably) is probably wise.